### DISCUSSION TITLE
[@types/ember] Wrong types for @ember/string

### DISCUSSION THREAD

**Discussion by kas1337** [DefinitelyTyped/DefinitelyTyped#67298] [Issues with a @types package]
Since @ember/string@3.1.1 the type definitions are not valid anymore.

```
node_modules/@types/ember/index.d.ts:389:41 - error TS2339: Property 'loc' does not exist on type 'typeof import("[...]/ember/node_modules/@ember/string/index")'.

389         const loc: typeof EmberStringNs.loc;
```
Could you update the types please?

[a comment by an automated account is not shown]

**Reply by chriskrycho** [reply]
@kas feel free to open a pull request to that effect, and a maintainer will help you get it landed! Or, better, if you can update to using the types shipped natively with the `@ember/string` package if you are on Ember v5 or later, that’ll be even better!
